### ISSUE TITLE
Icons (graphics) missing in the interface

### ISSUE THREAD

**Issue by fklemme** [Razviar/mtgap#367]
**Detailed context:**
- Link to your MTGA Pro account: mtga.cc/u/Kruecke
- Screenshots of issue if applicable: See bottom left corner in the screenshot below.
![image](https://github.com/Razviar/mtgap/assets/5053254/29ac120d-766d-479b-85fb-7fb4c53fdaa2)

**Your Desktop:**
 - OS: Windows 11
 - App type: Standalone (from Github)
 - App Version: 2.2.26 (latest)

**Describe the bug**
Icons are not shown/loaded in the Tracker interface and in the overlay. Only small, colored boxes are shown instead.
I tried adding the app executable as exceptions to Windows Defender, and I also wiped the config. Nothing helped.

**To Reproduce**
1. Install Tracker following the instructions in the README.
2. That's it. The icons are missing.

**Comment by fklemme** [comment]
![image](https://github.com/Razviar/mtgap/assets/5053254/67983870-5b1a-4368-9e9b-dd9e6e1216da)
Example of the overlay.

**Comment by Bl00dm0ney** [comment]
I created an account and installed the tracker for the first time earlier, and experienced the exact same missing icons. 

mtga.cc/u/Bloodmoney
Win10
App Version 2.2.26

**Comment by Day0fJustice** [comment]
I also installed just today and was experiencing this bug immediately upon use.  The boxes replacing icons is also present on the app in Windows, not only the in game overlay.
![image](https://github.com/Razviar/mtgap/assets/130005943/89c6db67-3a51-493f-842e-89ebd5f5ce02)

Windows 11
App v 2.2.26

**Comment by fizzikzz** [comment]
same thing here:
![image](https://github.com/Razviar/mtgap/assets/16635410/3d429967-2e3f-4106-80c1-a6163cb5df14)

**Comment by BrettMoan** [comment]
I'm on windows, and I personally resolved this issue by downloading fonts manually.

to do so i searched for "font download <>" where `<>`  is each of the fonts listed in the code base here https://github.com/search?q=repo%3ARazviar%2Fmtgap+font-family&type=code.


what I did, Use at your own risk:

I'm guessing the icons at the bottom of the initial screen are from font_awesome_5_brands. i personally got that from https://fonts2u.com/font-awesome-5-brands-regular.font

I also added the Titillium_Web which i think i found on google fonts.
